src/transformers/models/detr/feature_extraction_detr.py

```diff
@@ -0,0 +1,803 @@
+# coding=utf-8
+# Copyright 2021 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Feature extractor class for DETR."""
+
+import io
+from collections import defaultdict
+import pathlib
+from typing import Dict, List, Optional, Union
+
+import numpy as np
+from PIL import Image
+
+from ...feature_extraction_utils import BatchFeature, FeatureExtractionMixin
+from ...file_utils import TensorType, is_flax_available, is_tf_available, is_torch_available
+from ...image_utils import ImageFeatureExtractionMixin, is_torch_tensor
+from ...utils import logging
+from .modeling_detr import box_cxcywh_to_xyxy
+
+
+if is_torch_available():
+    import torch
+    import torch.nn.functional as F
+
+logger = logging.get_logger(__name__)
+
+
+# inspired by https://github.com/facebookresearch/detr/blob/master/util/box_ops.py
+def box_xyxy_to_cxcywh(x):
```

Comment:
The name is obscure for anyone who has never done object detection. Something like `corners_to_center_format` would be way more understandable and the docstring can precise more what "corners" format and "center" format are.

Reply:
Ok, renamed it.